src/main/kotlin/no/nav/klage/oppgave/service/unleash/ByEnhetStrategy.kt

```diff
@@ -37,7 +37,8 @@ class ByEnhetStrategy(val saksbehandlerRepository: SaksbehandlerRepository) : St
         }
 
     private fun getSaksbehandlersEnheter(unleashContext: UnleashContext) =
-        saksbehandlerRepository.getTilgangerForSaksbehandler(unleashContext.userId.get()).enheter.asList().map { it.enhetId }
+        saksbehandlerRepository.getTilgangerForSaksbehandler(unleashContext.userId.get()).enheter.asList()
```

Comment:
Er det en grunn til å gjøre om fra array til list før map?

Reply:
Godt spørsmål. Da jeg skrev den koden, så syntes jeg IntelliJ fortalte meg at array ikke hadde noen map funksjon, men det var åpenbart bare tull, for det fungerte fint nå.. Det hender at IntelliJ lurer meg litt...